src/common.h

```diff
@@ -162,10 +158,16 @@ struct Location {
       size_t offset;
     };
   };
+
+
+  Location() {
+    this->line = 0;
```

Comment:
initialize `filename` too. Also style is to use `:line(0), first_column(0),` etc